package.json

```diff
@@ -29,6 +29,7 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
     "@emotion/babel-preset-css-prop": "^10.0.27",
+    "@guardian/types": "guardian/types#tw-add-ophan-types",
```

Comment:
When this branch is merged we can create a new release of `@guardian/types` and use that.